app/helpers/twitter_intent.rb

```diff
@@ -0,0 +1,17 @@
+module ExercismWeb
+  module Helpers
+    module TwitterIntent
+      def twitter_share(opts = {})
```

Comment:
You could use optional keyword arguments here `def twitter_share(text: nil, url: nil)` and make it more explicit what arguments you're expecting.